README.md

```diff
@@ -518,6 +522,14 @@ sbuild --dist=<target> ../ubuntu-advantage-tools_*.dsc
 sbuild-launchpad-chroot create --architecture="riscv64" "--name=focal-riscv64" "--series=focal
 ```
 
+> Note: Every so often, it is recommended to update your chroots.
+> ```bash
+> # to update a single chroot
+> sudo sbuild-launchpad-chroot update -n ua-xenial-amd64
+> # this script can be used to update all chroots
+> sudo env PATTERN=\* sh /usr/share/doc/sbuild/examples/sbuild-debian-developer-setup-update-all
```

Comment:
Nit: `sudo` accepts env variables (`sudo VAR=value <command>`), so we don't need `env`.

Reply:
TIL! Thanks! Will fix this then merge